src/Symfony/Component/ErrorRenderer/Exception/FlattenException.php

```diff
@@ -374,3 +374,8 @@ public function getAsString()
         return rtrim($message);
     }
 }
+
+// to be removed in 5.0
+if (!class_exists(\Symfony\Component\Debug\Exception\FlattenException::class, false)) {
+    class_alias(FlattenException::class, \Symfony\Component\Debug\Exception\FlattenException::class);
```

Comment:
Btw, someone doing this `FlattenException::create()` using the legacy one will break things. I guess we should revert the `create()` method in the new class?